File: src/main/java/postBox/Box5.java
package postBox;

public class Box5 extends abstractBox {

    public Box5(Float length, Float width, int tall) {
        super(length, width, tall);
    }

    @Override
    public boolean validate(float length, float width, int tall) {
        if(check(length, width, tall)) {
            System.out.println("Box5");
            return true;
        }
        return false;
    }
}

File: src/main/kotlin/postBoxK/Box5.kt
package postBoxK

class Box5(length: Float, width: Float, tall: Int) : abstractBox(length, width, tall){

    override fun validate(length: Float, width: Float, tall: Int): Boolean =
        if(check(length, width, tall)){
            println("Box5")
            true
        }else
            false
    
}